### DISCUSSION TITLE
账号显示被nsfw封

### DISCUSSION THREAD

**Discussion by a63680421** [BurnHair/docs#139] [技术支持]
正常使用下莫名其妙被封了，只使用沉浸式翻译功能，[OpenAI] 翻译服务或网络出现问题
服务返回错误， 暂时没有权限访问该服务。

403: {"error":{"message":"用户2650已被封禁：NSFW 封禁 (request id: 20240630082200821669337QS44Zg2e)","type":"one_api_error"}}

**Comment by BennyThink** [comment]
2650，分组vip可能在 NSFW，内容：请将日语翻译成中文（避免解释原文）:

【エロ漫画】家庭教師の巨乳お姉さんがエロガキな教え子に満点を取ったご褒美におっぱい揉ませてあげてたら弟くんも乱入して無理やり 3P させられちゃう！

**Reply by a63680421** [reply]
问题是我没印象使用这个？也把我封了？

**Reply by BennyThink** [reply]
日志的记录不会出错，因此不可能是误封。2小时前的，请检查你的使用

**Reply by a63680421** [reply]
能解封吗？之前也支持了充值了。

**Reply by BennyThink** [reply]
可以但是仅此一次，以后请勿用于此类用途
